src/components/common/ImageContainer/index.tsx

```diff
@@ -0,0 +1,44 @@
+import Image, { ImageProps } from 'next/image';
+
+type ImageType = 'profile' | 'small' | 'medium' | 'full';
+
+interface ImageContainerProps {
+  type: ImageType;
+  src: string;
+}
+
+interface ImageAttributes extends Pick<ImageProps, 'width' | 'height'> {
+  radius: string;
+}
```

Comment:
```suggestion
interface ImageAttributes extends Omit<ImageProps, "width" | "height"> {
  radius: string;
}
```

next image의 타입에서 width, height를 제외한 모든 타입을 받아야해서 Pick대신 Omit을 사용하는게 좋을 것 같아요!
Omit은 어떤 타입 중에서 특정 몇몇개를 제외하겠다는 의미입니다
Pick은 반대로 어떤 타입 중에서 특정 몇몇개만 사용하겠다는 의미

Reply:
아하아하 반대로 이해하고 사용했네요

Reply:
> next image의 타입에서 width, height를 제외한 모든 타입을 받아야해서 Pick대신 Omit을 사용하는게 좋을 것 같아요! Omit은 어떤 타입 중에서 특정 몇몇개를 제외하겠다는 의미입니다 Pick은 반대로 어떤 타입 중에서 특정 몇몇개만 사용하겠다는 의미
`ImageAttributes` 에 `width`, `height` 를 추가를 해야합니다! Omit을 사용하면 제거가 되는데 사용하는게 맞을까욤,,!?

Reply:
ㅈㅅㅈㅅ 제가 잘못봤습니다
```suggestion
interface ImageAttributes extends Pick<ImageProps, 'width' | 'height'> {
  radius: string;
}
```

원래대로가 맞네요 ㅜㅠ